src/nginx/module.cc

```diff
@@ -609,6 +644,12 @@ Status ngx_http_esp_access_handler(ngx_http_request_t *r) {
     return Status(NGX_ERROR, "Missing esp request context.");
   }
 
+  if (ctx->api_basepath_rewrite_action == ApiBasepathRewriteAction::REJECT) {
+    ctx->status = Status(Code::NOT_FOUND, "Method does not exist.",
+                         Status::SERVICE_CONTROL);
```

Comment:
last argument should not be service_control

Reply:
Modified to Status::APPLICATION